engineering/project-template/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
why not pointing each `{{repo}}/CODE_OF_CONDUCT.md` to `https://github.com/src-d/guide/blob/master/.github/CODE_OF_CONDUCT.md` as we do with `CONTRIBUTING.md`, so that way we'll have only one version of that one?

Reply:
I think it's a good idea to avoid repetition, yeah